Sources/BBCodeParser.php

```diff
@@ -3691,9 +3694,9 @@ protected function closeTags(): void
 		}
 
 		foreach ($to_close as $tag) {
-			$this->message = substr($this->message, 0, $this->pos) . "\n" . $tag['after'] . "\n" . substr($this->message, $pos2 + 1);
+			$this->message = substr($this->message, 0, $this->pos) . "\n" . $tag['after'] . "\n" . substr($this->message, (int) ($pos2 + 1));
```

Comment:
Casting the result of the expression to int is unnecessary. The result of the arithmetic operation will already be an integer.